packages/babel-plugin-relay/__tests__/BabelPluginRelayMacro-test.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @emails oncall+relay
+ */
+
+'use strict';
+
+require('configureForRelayOSS');
+
+const babel = require('babel-core');
+
+describe('BabelPluginRelayMacro', () => {
+  test('works', () => {
+    const basic = `
+      'use strict';
+
+      const {graphql} = require('../../react-relay/modern/ReactRelayGraphQL.macro');
+      const CompatProfilePic = require('CompatProfilePic');
+
+      const CompatViewerQuery = graphql\`
+        query CompatViewerQuery($id: ID!, $scale: Float = 1.5) {
+          node(id: $id) {
+            ... on User {
+              id
+              ...CompatProfilePic_user
+            }
+          }
+        }
+      \`;
+    `;
+    const {code} = babel.transform(basic, {
+      plugins: ['babel-macros'],
+      filename: __filename,
+      compact: false,
+      parserOpts: {plugins: ['jsx']},
+      babelrc: false,
+    });
+    expect(code).toMatchSnapshot();
```

Comment:
Have you considered using [babel-plugin-tester](https://github.com/babel-utils/babel-plugin-tester)? It reduces the amount of boilerplate and the snapshot is a bit more insightful. [Here's an example with a macro](https://github.com/kentcdodds/babel-plugin-codegen/blob/master/src/__tests__/macro.js) and [here are the snapshots](https://github.com/kentcdodds/babel-plugin-codegen/blob/master/src/__tests__/__snapshots__/macro.js.snap).